index.js

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+const { headerTemplate, userTemplate } = require('./template');
+
+const pkg = require(path.resolve(process.cwd(), 'package.json'));
+
+function main(
+  file = 'CONTRIBUTORS.md',
+  owner = 'the-after-hours',
+  repo = 'contri-credit'
+) {
+  process.stdout.write('Getting contributor information...\n');
+
+  axios({
+    baseURL: 'https://api.github.com/',
+    method: 'GET',
+    url: `/repos/${owner}/${repo}/contributors`,
+    headers: {
+      'Cache-Control': 'no-cache',
+      'User-Agent': owner
+    }
+  })
+    .catch(function(error) {
+      process.stderr.write(`
+        Unable to request contirbutors.\n
+        Please ensure you have set up your package.json with the correct configurations
+        See our readme at https://github.com/the-after-hours/contri-credit
+        file: ${file}
+        owner: ${owner}
+        repo: ${repo}\n
+      `);
+
+      process.stderr.write(error);
+    })
+    .then(function(response) {
+      process.stdout.write(
+        'Reponse received. Generating contributor file...\n'
+      );
+
+      const userList = response.data;
+
+      const writeStream = fs.createWriteStream(file);
+
+      if (repo === 'contri-credit') {
```

Comment:
Maybe I'm just really tired, but they way this seems to be coding (you setting repo = 'contri-credit' in ln 13, this will always be true

Reply:
I think this can be improved , because we’ll probably throw an error before we get here. But ‘repo’ is grabbed from the local package json where this is ran. If they add ‘contri-credit’ without anything else, defaults get set. But if they don’t even add the contri-credit property , this will err before we get here.

Reply:
Right... and it erring out is not a problem. This should be good for now.